components/Account.php

```diff
@@ -58,7 +58,7 @@ public function defineProperties()
 
     public function getRedirectOptions()
     {
-        return [''=>'- refresh page -', '0' => '- no redirect -'] + Page::sortBy('baseFileName')->lists('baseFileName', 'baseFileName');
+        return [''=>'- refresh page -', '0' => '- no redirect -', 'intended' => '- intended url -'] + Page::sortBy('baseFileName')->lists('baseFileName', 'baseFileName');
```

Comment:
please put each option on its own line.

Reply:
Surely! I apologize but this is current file indentation. Is this ok?

```php 
return [
    '' => '- refresh page -',
    '0' => '- no redirect -',
    'intended' => '- intended url -',
] + Page::sortBy('baseFileName')->lists('baseFileName', 'baseFileName');
```